frontend/components/QuotesSlideshow.js

```diff
@@ -0,0 +1,206 @@
+            quotes.quoteBlock,
+            quotes.quoteBlockCareers,
+            customSlideStyle,
+          ].join(" ")}
+        >
+          {displaySlideInfo(key, quotes.careersRoleColour)}
+        </div>
+      );
+    } else if (data[key][1].includes("Competitions")) {
+      quoteBlock = (
+        <div
+          className={[
+            quotes.quoteBlock,
+            quotes.quoteBlockCompetitions,
+            customSlideStyle,
+          ].join(" ")}
+        >
+          {displaySlideInfo(key, quotes.competitionsRoleColour)}
+        </div>
+      );
+    } else if (data[key][1].includes("Externals")) {
+      quoteBlock = (
+        <div
+          className={[
+            quotes.quoteBlock,
+            quotes.quoteBlockExternals,
+            customSlideStyle,
+          ].join(" ")}
+        >
+          {displaySlideInfo(key, quotes.externalsRoleColour)}
+        </div>
+      );
+    } else if (data[key][1].includes("HR")) {
+      quoteBlock = (
+        <div
+          className={[
+            quotes.quoteBlock,
+            quotes.quoteBlockHR,
+            customSlideStyle,
+          ].join(" ")}
+        >
+          {displaySlideInfo(key, quotes.hrRoleColour)}
+        </div>
+      );
+    } else if (data[key][1].includes("Publications")) {
+      quoteBlock = (
+        <div
+          className={[
+            quotes.quoteBlock,
+            quotes.quoteBlockPublications,
+            customSlideStyle,
+          ].join(" ")}
+        >
+          {displaySlideInfo(key, quotes.publicationsRoleColour)}
+        </div>
+      );
+    } else if (data[key][1].includes("Marketing")) {
+      quoteBlock = (
+        <div
+          className={[
+            quotes.quoteBlock,
+            quotes.quoteBlockMarketing,
+            customSlideStyle,
+          ].join(" ")}
+        >
+          {displaySlideInfo(key, quotes.marketingRoleColour)}
+        </div>
+      );
+    } else if (data[key][1].includes("IT")) {
+      quoteBlock = (
+        <div
+          className={[
+            quotes.quoteBlock,
+            quotes.quoteBlockIT,
+            customSlideStyle,
+          ].join(" ")}
+        >
+          {displaySlideInfo(key, quotes.itRoleColour)}
+        </div>
+      );
+    } else if (data[key][1].includes("Media")) {
+      quoteBlock = (
+        <div
+          className={[
+            quotes.quoteBlock,
+            quotes.quoteBlockMedia,
+            customSlideStyle,
+          ].join(" ")}
+        >
+          {displaySlideInfo(key, quotes.mediaRoleColour)}
+        </div>
+      );
+    } else {
+      quoteBlock = (
+        <div
+          className={[
+            quotes.quoteBlock,
+            quotes.quoteBlockDefault,
+            customSlideStyle,
+          ].join(" ")}
+        >
+          {displaySlideInfo(key, quotes.defaultRoleColour)}
+        </div>
+      );
+    }
+    return quoteBlock;
+  };
+
+  const displaySlideInfo = (key, colorClass) => {
+    return (
```

Comment:
Remove extraneous whitespace for quotes slideshow

![image](https://user-images.githubusercontent.com/80564580/174700095-d5465e49-009a-4461-a564-e03b1641113b.png)

Reply:
done HEHE